src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,57 @@
 package mate.jdbc;
 
+import java.util.ArrayList;
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector INJECTOR = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        DriverService driverService = (DriverService) INJECTOR.getInstance(DriverService.class);
+
+        Driver petro = new Driver("Petro", "4131");
+        Driver ivan = new Driver("Ivan", "3122");
+        Driver createdPetro = driverService.create(petro);
+        Driver createdIvan = driverService.create(ivan);
+        Driver newPetro = driverService.get(createdPetro.getId());
+        Driver newIvan = driverService.get(createdIvan.getId());
+        System.out.println("Petro: " + newPetro);
+        System.out.println("Ivan: " + newIvan);
+        List<Driver> puntoDrivers = new ArrayList<>();
+        puntoDrivers.add(createdIvan);
+        puntoDrivers.add(createdPetro);
+        ManufacturerService manufacturerService
```

Comment:
Please add sometimes an empty lines for divide some code blocks from each other and increase code readability
```suggestion
        puntoDrivers.add(createdPetro);
        
        ManufacturerService manufacturerService
```